### ISSUE TITLE
@FilterableField() with nullable:true throws error

### ISSUE THREAD

**Issue by johannesschobel** [doug-martin/nestjs-query#20]
Dear @doug-martin ,

a 
```ts
@FilterableField({
  description: 'firstname of the user',
  nullable: true
})
firstname?: string
```
throws an error when requesting this field in a query (and if it is actually `null`).

Changing from `@FilterableField()` to `@Field()` removes this issue, however, it is not possible to search by this field any more..

All the best

**Comment by doug-martin** [comment]
Thanks @johannesschobel! Ill start fixing the issues tonight. 

Thank you reporting the issues you find I really appreciate it!

**Comment by doug-martin** [comment]
Feel free to make a PR on this one also if you want.

**Comment by johannesschobel** [comment]
thank you for creating this awesome package!

**Comment by doug-martin** [comment]
Issued pr #21 to fix this.

**Comment by doug-martin** [comment]
Published! The version with the fix is `v0.2.1`

**Comment by johannesschobel** [comment]
awesome support.. works like a charm!
